src/LayerPlan.cpp

```diff
@@ -774,6 +774,31 @@ void LayerPlan::writeGCode(GCodeExport& gcode)
                 }
                 gcode.writeTemperatureCommand(prev_extruder, prev_extruder_temp, wait);
             }
+
+            if (!getPrimeTowerIsPlanned())
```

Comment:
This code should be removed and the prime poop should instead be performed on the first layer.
There seems to be a bug there, cause the prime poop should already always be planned on the first layer...